src/accessors.jl

```diff
@@ -122,13 +122,30 @@ get_commitment(system::SystemRT) = system.generator_status.commitment
 "Returns time series data of generator regulation commitment status in each hour"
 get_regulation_commitment(system::SystemRT) = system.generator_status.regulation_commitment
 
+"Returns a `Dictionary` of `Technology`s indexed by Generator ID"
+get_technologies(system::System) = map(gen -> gen.technology, get_generators(system))
```

Comment:
What do you think of also adding a `get_fuel_type` function?
```suggestion
get_technologies(system::System) = map(gen -> gen.technology, get_generators(system))
"Returns a `Dictionary` of `Symbol`s describing generator fuel types indexed by Generator ID"
get_fuel_type(system::System) = map(gen -> gen.technology.fuel_type, get_generators(system))
```

Reply:
Good idea!